tests/test_pptx/test_pptx_integration.py

```
@@ -0,0 +1,491 @@
+
+            # Find table elements
+            table_elements = [e for e in table_slide.elements if isinstance(e, PowerPointTableElement)]
+
+            if table_elements:  # Only test if we successfully found/converted table elements
+                table_element = table_elements[0]
+                api_client = PowerPointAPIClient()
+
+                # Test table resizing
+                try:
+                    table_element.resize(api_client, rows=4, cols=4)
+                except Exception as e:
```

Ruff (0.15.6)

[warning] Do not catch blind exception: `Exception`

(BLE001)